src/manager.S

```diff
@@ -139,7 +139,13 @@ add_compartment:
  * provided to it for use (i.e., PCC, DDC), such as by storing it in the memory
  * of another compartment. If the original compartment gets deleted, and a new
  * compartment is allocated in the same memory region, we have leaked a
- * capability referencing the new compartment. */
+ * capability referencing the new compartment.
+ * TODO currently, this function shifts subsequent data following a delete
+ * compartment meta-data in its place. It would be possible for a malicious
+ * compartment to inject valid malicious data in the space subsequent to this
+ * are of code, and somehow that data can be misused by the switcher. One
```

Comment:
"are of code" doesn't parse to me.

Reply:
Fixed in https://github.com/capablevms/cheri-morello-compartmentalisation/pull/2/commits/8c16868ac2a09e767d91288856584384dbb2e189.